.github/workflows/code-formatting-check.yml

```diff
@@ -0,0 +1,18 @@
+name: "Check Formatting"
```

Comment:
this is only talking or also block/pass check?

Reply:
It is outputing which files should be formated